src/test/java/com/easypost/EventTest.java

```diff
@@ -76,4 +82,73 @@ public void testRetrieve() throws EasyPostException {
         // Must compare IDs since can't do whole-object comparisons currently
         assertEquals(event.getId(), retrievedEvent.getId());
     }
+
+    /**
+     * Test retrieving all payloads.
+     *
+     * @throws EasyPostException when the request fails.
+     * @throws InterruptedException when vcr recording fails.
+     */
+    @Test
+    public void testEventRetrieveAllPayloads() throws EasyPostException, InterruptedException {
+        vcr.setUpTest("retrieve_all_payloads");
+
+        Map<String, Object> params = new HashMap<>();
+        params.put("url", Fixtures.webhookUrl());
+
+        Webhook webhook = vcr.client.webhook.create(params);
+        Map<String, Object> batchParams = new HashMap<>();
+
+        List<Object> shipments = new ArrayList<>();
+        shipments.add(Fixtures.basicShipment());
+        batchParams.put("shipments", shipments);
+        vcr.client.batch.create(batchParams);
```

Comment:
I'd like these to be paired up better. Put like-things together

```suggestion
        Map<String, Object> params = new HashMap<>();
        params.put("url", Fixtures.webhookUrl());
        Webhook webhook = vcr.client.webhook.create(params);
        
        List<Object> shipments = new ArrayList<>();
        shipments.add(Fixtures.basicShipment());

        Map<String, Object> batchParams = new HashMap<>();
        batchParams.put("shipments", shipments);
        vcr.client.batch.create(batchParams);
```

Reply:
resolved